src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js

```diff
@@ -465,6 +472,13 @@ class BasePaymentsPage extends React.Component {
                                     text={this.props.translate('paymentsPage.setDefaultConfirmation')}
                                 />
                             )}
+                            {isPayPalMeSelected && (
+                                <Button
+                                    onPress={() => this.navigateToAddPaypalRoute()}
```

Comment:
```suggestion
                                    onPress={this.navigateToAddPaypalRoute}
```

Reply:
Just a small styling fix, we can replace {() => this.navigateToAddPaypalRoute()} with {this.navigateToAddPaypalRoute}, otherwise looks good